<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserInfoConfig</class>
 <widget class="QDialog" name="UserInfoConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>400</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/bg/userInfoConfig.png);</string>
   </property>
   <widget class="QLineEdit" name="userNameEdit">
    <property name="geometry">
     <rect>
      <x>85</x>
      <y>128</y>
      <width>200</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/bg/content.png);
border-image: url(:/bg/content.png);</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>用户名</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="userNickNameEdit">
    <property name="geometry">
     <rect>
      <x>85</x>
      <y>160</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/bg/content.png);
border-image: url(:/bg/content.png);</string>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>昵称</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="userInfoEdit">
    <property name="geometry">
     <rect>
      <x>85</x>
      <y>205</y>
      <width>201</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/bg/content.png);
border-image: url(:/bg/content.png);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pswOldEdit">
    <property name="geometry">
     <rect>
      <x>395</x>
      <y>130</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/bg/content.png);
border-image: url(:/bg/content.png);</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>旧的密码</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pswNew1Edit">
    <property name="geometry">
     <rect>
      <x>395</x>
      <y>170</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/bg/content.png);
border-image: url(:/bg/content.png);</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>新的密码</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pswNew2Edit">
    <property name="geometry">
     <rect>
      <x>395</x>
      <y>210</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/bg/content.png);
border-image: url(:/bg/content.png);</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>确认密码</string>
    </property>
   </widget>
   <widget class="QToolButton" name="CloseWinBtn">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>60</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/bg/content.png);
border-image: url(:/bg/closeBtn.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
